src/Tizen.NUI.WindowSystem/src/public/QuickPanelClient.cs

```diff
@@ -260,6 +294,46 @@ public QuickPanelClient(TizenShell tzShell, Window win, Types type)
             if (width > height) _screenOrientation = Window.WindowOrientation.Landscape;
         }
 
+        /// <summary>
+        /// Creates a new Quickpanel Client handle.
+        /// </summary>
+        /// <param name="tzShell">The TizenShell instance.</param>
+        /// <param name="win">The window provider for the quickpanel service.</param>
+        /// <param name="type">The type of quickpanel service.</param>
+        /// <exception cref="Tizen.Applications.Exceptions.OutOfMemoryException">Thrown when the memory is not enough to allocate.</exception>
```

Comment:
would sound better: "when there is not enough memory (to allocate)"

Reply:
I have changed following your comments.